### DISCUSSION TITLE
Is it possible to support multiple languages?

### DISCUSSION THREAD

**Discussion by MephistoSatanDevil** [NearInfinityBrowser/NearInfinity#173] [Ideas & Feature Requests]
Is it possible to support multiple languages? Translations could be provided by the community.

**Comment by Argent77** [comment]
That would be theoretically possible but it's not really feasible. The UI uses countless instances of hardcoded strings. In many cases they are generated or assembled dynamically. It would be a monumental task to change that for the whole application.
